apps/azureserviceoperator-system/resources/resource-group.yaml

```diff
@@ -7,4 +7,9 @@ metadata:
     serviceoperator.azure.com/reconcile-policy: detach-on-delete
 spec:
   location: uksouth
-  azureName: ${NAMESPACE}-aso-${ENVIRONMENT}-rg
\ No newline at end of file
+  azureName: ${NAMESPACE}-aso-${ENVIRONMENT}-rg
+  tags:
+    builtFrom: https://github.com/hmcts/cnp-flux-config
+    businessArea: CFT
+    environment: development
+    exemptFromAutoLock: "true"
```

Comment:
should probably add the application tag too with the same fallback as in the other one?

Reply:
sure -done